packages/augur-ui/src/modules/auth/components/connect-dropdown/connect-dropdown.tsx

```diff
@@ -29,6 +29,8 @@ interface ConnectDropdownProps {
   userDefinedGasPrice: string;
   gasPriceSpeed: number;
   showAddFundsModal: Function;
+  universeSelectorModal: Function;
+  universe: Universe;
```

Comment:
Since now you only need universe name you dont have to pass the whole universe object